virtual_rainforest/models/abiotic/constants.py

```diff
@@ -163,3 +163,64 @@ class AbioticConsts(ConstantsDataclass):
 
     turbulence_sign: bool = True
     """Flag indicating if turbulence increases or decreases with height."""
+
+    canopy_temperature_ini_factor: float = 0.01
+    """Factor used to initialise canopy temperature as a function of air temperature and
+    absorbed shortwave radiation."""
+
+    light_extinction_coefficient: float = 0.01
+    """Light extinction coefficient for canopy, default = 0.01."""
+
+    latent_heat_flux_fraction: float = 0.5
+    """Fraction of incoming energy that is turned into latent heat flux, default = 0.5.
+    """
+
+    ground_heat_flux_fraction: float = 0.05
+    """Fraction of incoming energy that is turned into ground heat flux, default = 0.05.
+    """
+
+    gas_constant_water_vapor: float = 461.51
+    """Gas constant for water vapor, [J kg -1 K-1]."""
+
+    specific_heat_capacity_leaf: float = 2760.0
+    """Specific heat capacity of leaf, [J kg-1 K-1], default = 2760
+    :cite:p:`aston_heat_1985`.
+    """
+
+    leaf_heat_transfer_coefficient: float = 50.0
+    """Leaf heat transfer coefficient, [s^1/2 m^-1/2],
+    :cite:p:`linacre_determinations_1964`, default = 50.
+    """
+
+    stomatal_resistance: float = 200.0
+    """Stomatal resistance, default = 200."""
```

Comment:
I don't think you strictly need to add default values here, as we _should_ be able to document the defaults automatically. Though, if you find adding them here helpful to you then that's fine